README.md

```diff
@@ -125,15 +125,16 @@ so I suggest you define them in `~/.config/nvim/after/ftplugin/rust.lua`[^1]
 Example:
 
 ```lua
-local bufnr = vim.api.nvim_get_current_buf()
-vim.keymap.set(
-  "n", 
-  "<leader>a", 
-  function()
-    vim.cmd.RustLsp('codeAction') 
-  end,
-  { silent = true, buffer = bufnr }
-)
+vim.g.rustaceanvim = {
```

Comment:
issue: This snippet is an example for how to set keymaps in `~/.config/nvim/after/ftplugin/rust.lua`, which is sourced **after** the plugin initializes. Setting `vim.g.rustaceanvim` here would be useless, because it would be too late for it to be picked up.

Reply:
I see, I never used ftplugin. Isn't is a little special for the quick setup?

Reply:
It's a built-in neovim mechanism.
And it's simpler than putting the keymaps in an `on_attach` function.

If you're not familiar with the feature, I recommend you check it out.

Instead of setting up multiple filetype autocmds (each of which adds overhead), with ftplugin, you use an existing autocmd that is built into neovim.